inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -85,16 +91,109 @@ class XmlSerializer : public ngraph::AttributeVisitor {
+            if (const auto& a = ngraph::as_type<ngraph::AttributeAdapter<std::vector<std::shared_ptr
+                        <ngraph::op::util::SubGraphOp::InputDescription>>>>(&adapter)) {
+                pugi::xml_node port_map = m_xml_node.parent().child("port_map");
+                if (!m_xml_node.parent().child("port_map")) {
+                    port_map = m_xml_node.parent().insert_child_before("port_map", m_xml_node.parent().first_child());
+                }
+
+                for (const auto& input_description : a->get()) {
+                    pugi::xml_node input = port_map.append_child("input");
+                    input.append_attribute("external_port_id").set_value(input_description->m_input_index);
+                    input.append_attribute("internal_layer_id").set_value(parameter_mapping[input_description->m_body_parameter_index].c_str());
+
+                    if (auto slice_input = as_type_ptr<ngraph::op::util::SubGraphOp::SliceInputDescription>(input_description)) {
+                        input.prepend_attribute("axis").set_value(slice_input->m_axis);
+                        if (slice_input->m_start) {
+                            input.append_attribute("start").set_value(slice_input->m_start);
+                        }
+                        if (slice_input->m_end != -1) {
+                            input.append_attribute("end").set_value(slice_input->m_end);
+                        }
+                        if (slice_input->m_stride != 1) {
+                            input.append_attribute("stride").set_value(slice_input->m_stride);
+                        }
+                        if (slice_input->m_part_size != 1) {
+                            input.append_attribute("part_size").set_value(slice_input->m_part_size);
+                        }
+                    } else if (auto merged_input = as_type_ptr<ngraph::op::util::SubGraphOp::MergedInputDescription>(input_description)) {
+                        pugi::xml_node back_edges = m_xml_node.parent().child("back_edges");
+                        if (!back_edges) {
+                            back_edges = m_xml_node.parent().insert_child_after("back_edges", port_map);
+                        }
+                        pugi::xml_node edge = back_edges.append_child("edge");
+                        edge.append_attribute("from-layer").set_value(result_mapping[merged_input->m_body_value_index].c_str());
+                        edge.append_attribute("to-layer").set_value(parameter_mapping[merged_input->m_body_parameter_index].c_str());
+                    }
+                }
+            } else if (const auto& a = ngraph::as_type<ngraph::AttributeAdapter<std::vector<std::shared_ptr
+                        <ngraph::op::util::SubGraphOp::OutputDescription>>>>(&adapter)) {
+                pugi::xml_node port_map = m_xml_node.parent().find_child([](pugi::xml_node node) {return strcmp(node.name(), "port_map") == 0;});
```

Comment:
.parent().child("port_map");

Reply:
will be removed in Loop serialization as it will modify on_adapter() for input/output description.